cts-java/src/test/java/org/ga4gh/cts/api/endpoints/RawEndpointsIT.java

```diff
@@ -0,0 +1,123 @@
+package org.ga4gh.cts.api.endpoints;
+
+import com.mashape.unirest.http.Unirest;
+import com.mashape.unirest.http.exceptions.UnirestException;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import org.ga4gh.ctk.transport.TransportUtils;
+import org.ga4gh.ctk.transport.URLMAPPING;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.net.HttpURLConnection;
+
+import static org.assertj.core.api.StrictAssertions.assertThat;
+
+/**
+ * Tests for the properties of the raw server endpoints.
+ *
+ * @author Herb Jellinek
+ */
+@RunWith(JUnitParamsRunner.class)
+@Category(EndpointsTests.class)
+public class RawEndpointsIT {
+
+    private static final URLMAPPING urls = URLMAPPING.getInstance();
+
+    /**
+     * Given a partial URL, use {@link TransportUtils#makeUrl(String, String)} to combine it with
+     * the root URL and return the result.
+     * @param partialUrl a partial URL
+     * @return a full URL including the root URL
+     */
+    private static String makeFullUrl(String partialUrl) {
+        return TransportUtils.makeUrl(urls.getUrlRoot(), partialUrl);
+    }
+
+    /**
+     * Test that the basic server verbs/methods work as expected.
+     *
+     * @param fullUrl the URL to test
+     */
+    private void testHttpMethods(String fullUrl) throws UnirestException {
```

Comment:
Neither of those methods is a (JUnit) test: they're both utility methods called by test methods defined elsewhere in the file.

How to tell what's a test method from what's not?  Tests must be:
- annotated with the `@Test` tag.
- declared as `public void`.
- declared to throw some kind of exception, or subclass of exception.

Sorry if the name `testHttpMethods` led you to think otherwise.  If I recall correctly, older versions of JUnit looked for `public void` methods with names beginning with `test`, which probably contributed to the confusion.

The only true JUnit tests in this file are `checkCallSetsRouting`, `checkVariantSearchMethods`, and `checkSearchRouting`.

I'm not sure I understand your comment about multiple lines - can you explain?